### DISCUSSION TITLE
Thoughts about scalability

### DISCUSSION THREAD

**Discussion by michielbdejong** [tosdr/edit.tosdr.org#987] [Ideas]
How can we do 10 or 100 times more reviews than we do now?

* The edit.tosdr.org site could be a bit faster, it's sometimes slowing down the reviewer/curator, especially things like loading the list of services
* We could reduce blocking friction like https://forum.tosdr.org/t/request-a-service-to-be-added/14
* We could automate more, like:
  - suggesting points based on ML
  - suggesting document URLs based on anchor tags from the home page
  - suggesting services by domain name, based on Alexa

See also https://github.com/tosdr/edit.tosdr.org/issues/926 for brainstorm about how to scale the tosback part of it.

**Comment by michielbdejong** [comment]
Maybe also automate that reviewers become curators, for instance if they have had 10 points approved by existing curators. If a spammer or hater were to slip through the net, we could manually downgrade them as well as the person who approved their bad points, or invent other loops for people to jump through if we feel it's necessary. Another option is to allow curators to explicitly vouch for others.

**Comment by JustinBack** [comment]
10 points seems a little low as its easy to get 10 points approved on the site in like 30 minutes to 1 hour? There should be more factors such as but not limited to:

- Account Age
- Amount of contributions (as you said above, I think a starting point of 30-40 is acceptable)

Maybe let a bot post users who have reached that limit in a hidden staff thread on the forums that the user is ready to review. (Discourse does this as well to speak of with the trust levels)
